base/applications/games/solitaire/solgame.cpp

```diff
@@ -553,13 +583,18 @@ void CARDLIBPROC DeckClickProc(CardRegion &stackobj, int iNumClicked)
 
         //remove the top 3 from deck
         activepile.Push(temp);
+
+        VisiblePileCards = numcards;
     }
 
     activepile.Print();
 
     pDeck->SetCardStack(cardstack);
     pPile->SetCardStack(pile);
 
+    // Fix for CORE-11148
```

Comment:
Not sure whether it's interesting to know in the code that it's a fix for that precise bug. This information should be instead conveyed in the commit log.

Reply:
Since it is not a workaround but an important line that was missing from the code (and caused the bug), I think this comment is not so important.

Reply:
ok :)

Reply:
Removed.